e2e/cypress/integration/plugins/marketplace/marketplace_spec.js

```diff
@@ -197,7 +198,7 @@ describe('Plugin Marketplace', () => {
             cy.get('#error_bar').should('not.be.visible');
 
             // * search should be visible
-            cy.findByPlaceholderText('Search Plugins').should('be.visible').click();
+            cy.findByPlaceholderText('Search Plugins').should('be.visible');
```

Comment:
This `click` was preventing the focus test from being meaningful, since it was already focussed manually. There are other failing e2e tests in my local sandbox, but they appear to be untouched by this change.

Reply:
I think this was there before we introduced the change to be focused.